internal/internal_task_handlers.go

```diff
@@ -1195,6 +1187,24 @@ func (w *workflowExecutionContextImpl) GetDecisionTimeout() time.Duration {
 	return time.Second * time.Duration(w.workflowInfo.TaskStartToCloseTimeoutSeconds)
 }
 
+func (w *workflowExecutionContextImpl) getWorkflowPanicIfIllegaleStatePanic() (*workflowPanicError, bool) {
+	if !w.isWorkflowCompleted || w.err == nil {
```

Comment:
`if !w.isWorkflowCompleted`
^ since this is a semantic change: are state-machine-panicked workflows _definitely_ completed?

Reply:
`w.err` is only set during [workflow completion](https://github.com/taylanisikdemir/cadence-client/blob/380c778f353b0cc476a5ad1c9c7a0d9949a4cf3f/internal/internal_task_handlers.go#L523). So I don't see the concern here